src/main.rs

```diff
@@ -22,6 +22,13 @@ fn main() {
         .help("The status code of the previously run command")
         .takes_value(true);
 
+    let job_arg = Arg::with_name("job")
+        .short("j")
+        .long("job")
+        .value_name("JOB")
+        .help("The number of currently running jobs")
+        .takes_value(0);
```

Comment:
Why the `.takes_value(0)`? 🤔 
afaik, `takes_value()` accepts a boolean value.

Reply:
You are right ... setting to `false` by default